### ISSUE TITLE
It looks like slop released a new version that isn't backwards compatible ...

### ISSUE THREAD

**Issue by harlanbarnes** [jimmycuadra/lita-keyword-arguments#1]
I use this extension in my rundeck handler and it got the exception below. I happened to notice that the `slop` rubygem had recently released a `4.0`and the lita-keyword-arguments' rubyspec was set to depend on `slop >= 3.5`. So I pinned slop to `3.5` and all was well again.

```
[2015-01-12 14:13:32 UTC] ERROR: Lita::Handlers::Rundeck crashed. The exception was:
undefined method `new' for Slop:Module
Full backtrace:
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments/parser.rb:36:in `parser'
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments/parser.rb:32:in `add_option'
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments/parser.rb:15:in `block in parse'
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments/parser.rb:15:in `each'
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments/parser.rb:15:in `parse'
/opt/lita/vendor/ruby/2.1.0/gems/lita-keyword-arguments-0.0.1/lib/lita/extensions/keyword_arguments.rb:9:in `call'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/handler/chat_router.rb:86:in `block in dispatch_to_route'
/usr/lib/ruby/2.1.0/set.rb:263:in `each_key'
/usr/lib/ruby/2.1.0/set.rb:263:in `each'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/handler/chat_router.rb:86:in `dispatch_to_route'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/handler/chat_router.rb:73:in `block in dispatch'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/handler/chat_router.rb:70:in `map'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/handler/chat_router.rb:70:in `dispatch'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/robot.rb:58:in `block in receive'
/usr/lib/ruby/2.1.0/set.rb:263:in `each_key'
/usr/lib/ruby/2.1.0/set.rb:263:in `each'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/robot.rb:55:in `map'
/opt/lita/vendor/ruby/2.1.0/gems/lita-4.0.4/lib/lita/robot.rb:55:in `receive'
/opt/lita/vendor/ruby/2.1.0/gems/lita-hipchat-2.1.1/lib/lita/adapters/hipchat/callback.rb:20:in `block in private_message'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/callbacks.rb:99:in `call'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/callbacks.rb:99:in `block in process'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/callbacks.rb:98:in `each'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/callbacks.rb:98:in `process'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/stream.rb:277:in `receive'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/streamparser.rb:74:in `block in parse'
/usr/lib/ruby/2.1.0/rexml/parsers/sax2parser.rb:142:in `call'
/usr/lib/ruby/2.1.0/rexml/parsers/sax2parser.rb:142:in `block in parse'
/usr/lib/ruby/2.1.0/rexml/parsers/sax2parser.rb:142:in `each'
/usr/lib/ruby/2.1.0/rexml/parsers/sax2parser.rb:142:in `parse'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/streamparser.rb:91:in `parse'
/opt/lita/vendor/ruby/2.1.0/gems/xmpp4r-0.5.6/lib/xmpp4r/stream.rb:75:in `block in start'
```

**Comment by jimmycuadra** [comment]
Yikes... Thanks for the report. I'll get a new plugin out that supports the latest Slop.

**Comment by jimmycuadra** [comment]
After spending some time on this, I don't think Slop 4.x is really usable at this point. It doesn't support all the features of 3.x and it doesn't add anything useful. I locked this plugin to use 3.x and released it as lita-keyword-arguments 1.0.
